### PR TITLE
Publish internal core patches

### PR DESCRIPTION
**Pull request by lbudai**

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by lbudai** [comment]
Do not merge this PR, we will update it.

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

File: modules/affile/affile-dest.c [hunk 1]
```diff
@@ -265,6 +266,9 @@ affile_dw_deinit(LogPipe *s)
 static void
 affile_dw_queue(LogPipe *s, LogMessage *lm, const LogPathOptions *path_options, gpointer user_data)
 {
+  if (!affile_dw_queue_enabled_for_msg(lm))
+    return;
```
Comment (MrAnno) [inline modules/affile/affile-dest.c]: Shouldn't we drop the message here?

Comment (lbudai) [inline modules/affile/affile-dest.c]: yeah, it would be better to move here the dropping

Comment (MrAnno) [inline modules/affile/affile-dest.c]: Thanks. :)

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by lbudai** [comment]
I think this is ready to merge(on our handover meeting we could merge it)

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Pull request merged**